pkg/collector/corechecks/snmp/devicecheck/devicecheck_test.go

```diff
@@ -323,8 +324,64 @@ community_string: public
 	sender.AssertMetric(t, "Gauge", "snmp.devices_monitored", float64(1), "", []string{"snmp_device:1.2.3.4"})
 
 	// with hostname
-	deviceCk.config.UseDeviceIDAsHostname = true
```

Comment:
Why do we need to remove this ? it changes a bit the meaning of this part of the test `// with hostname`.

Reply:
I removed it as I found it misleading. This test does not make any use of `UseDeviceIDAsHostname`.